conf/zeppelin-site.xml.template

```diff
@@ -219,5 +219,11 @@
   <description>Anonymous user allowed by default</description>
 </property>
 
+<property>
+  <name>zeppelin.websocket.maxTextMessageSize</name>
```

Comment:
nit: should this be `zeppelin.websocket.max.text.message.size` for naming consistency?